drivers/start_date_dq/app/models/start_date_dq/report.rb

```diff
@@ -0,0 +1,80 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module StartDateDq
+  class Report
+    include ActionView::Helpers::NumberHelper
+    include ArelHelper
+    include Filter::FilterScopes
+
+    attr_reader :filter
+
+    def initialize(user_id, filter = nil)
+      @filter = filter || default_filter(user_id)
+    end
+
+    def default_filter(user_id)
+      Filters::FilterBase.new(user_id: user_id, start: '2018-01-01')
+    end
+
+    def self.viewable_by(user)
+      GrdaWarehouse::WarehouseReports::ReportDefinition.where(url: url).
+        viewable_by(user).exists?
+    end
+
+    def self.url
+      'start_date_dq/warehouse_reports/reports'
+    end
+
+    def title
+      'Approximate Start Date Data Quality'
+    end
+
+    def column_names
+      [
+        'Days Between DateToStreet and EntryDate',
+        'DateToStreetESSH',
+        'Entry Date',
+        'Personal ID',
+        'Project',
+        'Project Type',
+      ]
+    end
+
+    def column_values(row)
+      date_to_street = row.enrollment.DateToStreetESSH
+      entry_date = row.enrollment.EntryDate
+      {
+        'days_between' => (entry_date - date_to_street).to_i,
+        'date_to_street' => date_to_street,
+        'entry_date' => entry_date,
+        'personal_id' => row.enrollment.PersonalID,
+        'project_name' => GrdaWarehouse::Hud::Project.confidentialize(name: row.project&.name),
+        'project_type' => HUD.project_type_brief(row.project_type),
+      }
```

Comment:
Another really tiny thing, but these can be symbols, like so:
```suggestion
      {
        days_between: (entry_date - date_to_street).to_i,
        date_to_street: date_to_street,
        entry_date: entry_date,
        personal_id: row.enrollment.PersonalID,
        project_name: GrdaWarehouse::Hud::Project.confidentialize(name: row.project&.name),
        project_type: HUD.project_type_brief(row.project_type),
      }
```